data/json/monsters/zed_misc.json

```diff
@@ -499,6 +503,8 @@
     "bleed_rate": 50,
     "vision_day": 7,
     "vision_night": 4,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
```

Comment:
Needs a new weakpoint set that doesn't include a head critical.

Reply:
Done